packages/ng/date2/date-range-input/date-range-input.component.html

```diff
@@ -25,7 +25,7 @@
 					(click)="editedField.set(0);openPopover(popoverRef, 'start')"
 					(keydown.arrowDown)="arrowDown(popoverRef, 'start')"
 					(keydown.escape)="popoverRef.close()"
-					[disabled]="disabled"
+					[disabled]="disabled()"
```

Comment:
erf that could have been caught by a lint check :/

Reply:
> We might need a linter rule or something to prevent this in the future.

Yup :/